src/nats/mod.rs

```diff
@@ -35,6 +37,13 @@ impl NatsServerCmd {
         self.pass = Some(password.to_owned());
         self
     }
+
+    // not having docs here is currently allowed to address the missing docs problem one place at a time. Helping us by documenting just one of these places helps other devs tremendously
+    #[allow(missing_docs)]
```

Comment:
Please, just cover the method with documentation

We allow `missing_docs` only for existing (old, not covered) code. For all new code we add the documentation, since the target is to cover all parts